### ISSUE TITLE
mqtt - Port from the URL is not being respected.

### ISSUE THREAD

**Issue by wendyOn** [infinyon/fluvio-connectors#164]
Port is fixed to 1883, should use port from URL

[MqttOptions::new](https://docs.rs/rumqttc/latest/rumqttc/struct.MqttOptions.html#method.new)
```rust
MqttOptions::new(client_id, host, port)
```

src/main.rs
```rust
#[structopt(short, long)]
mqtt_url: String,

/// -- clip --

let mut mqttoptions = MqttOptions::new(client_id, mqtt_url, 1883);
```

**Comment by nacardin** [comment]
Should also respect url protocol, `mqtt` or `mqtts`.

**Comment by simlay** [comment]
Closed in #179.
